shared/src/main/java/net/skinsrestorer/shared/utils/log/SRLogger.java

```diff
@@ -60,6 +60,14 @@ public void debug(String message) {
         debug(SRLogLevel.INFO, message);
     }
 
+    public void debug(String message, Throwable thrown) {
+        debug(SRLogLevel.WARNING, message, thrown);
+    }
+
+    public void debug(Throwable thrown) {
+        debug(SRLogLevel.WARNING, "Received error", thrown);
```

Comment:
is suffix space not missing?
example: `Received error:(space) thrown`

Reply:
I think that's not how it would look like. 
More like a
```
Received error
stacktrace
Error: This is the error message from the object
stacktrace
```